docs/articles/documentation/test-api/built-in-waiting-mechanisms.md

```diff
@@ -113,7 +119,26 @@ test('My test', async t => {
 });
 ```
 
-## Waiting for Redirects
+## Wait Mechanism for XHR and Fetch Requests
+
+Before TestCafe executes a test action, it waits for XHR and fetch requests to complete within **3** seconds. After TestCafe receives the responses, or the timeout exceeds, the test continues.
```

Comment:
TestCafe waits **3** seconds for XHR and fetch requests to complete before it executes a test action. The test continues after responses are received or the timeout is exceeded.